en/migrations.rst

```diff
@@ -129,21 +140,25 @@ The following ``Bake`` command would create a migration to add a ``products`` ta
         Creating file /home/user/Work/php/cakeblog/config/Migrations/20160121163249_CreateProducts.php
         Wrote `/home/user/Work/php/cakeblog/config/Migrations/20160121163249_CreateProducts.php`
 
-.. note::
+You can also use the ``underscore_form`` as the name for your migrations
+i.e. create_products::
+
+        $ bin/cake bake migration create_products name:string description:text created modified
+    
+        Welcome to CakePHP v3.0.13 Console
+        ---------------------------------------------------------------
+        App : src
+        Path: /home/user/Work/php/cakeblog/src/
+        ---------------------------------------------------------------
+        
+        Creating file /home/user/Work/php/cakeblog/config/Migrations/20160121164955_create_products.php
+        Wrote `/home/user/Work/php/cakeblog/config/Migrations/20160121164955_create_products.php`
+
+.. versionadded:: cakephp/migrations 1.5.2
 
-        You can also use the ``underscore_form`` as the name for your migrations
-        i.e. create_products::
-
-                $ bin/cake bake migration create_products name:string description:text created modified
-            
-                Welcome to CakePHP v3.0.13 Console
-                ---------------------------------------------------------------
-                App : src
-                Path: /home/user/Work/php/cakeblog/src/
-                ---------------------------------------------------------------
-                
-                Creating file /home/user/Work/php/cakeblog/config/Migrations/20160121164955_create_products.php
-                Wrote `/home/user/Work/php/cakeblog/config/Migrations/20160121164955_create_products.php`
+    Camelizing the file name of the initial migration was introduced in v1.5.2 of the
+    `migrations plugin <https://github.com/cakephp/migrations/>`_. This version of
+    the plugin is only available with a release of CakePHP >= to 3.1.
```

Comment:
And here, add a note that prior to this version of the plugin, the name would be in the underscored form.